recipes/tkwant/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+platform='unknown'
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
+   cp $RECIPE_DIR/build_mac.conf build.conf
+else
+   cp $RECIPE_DIR/build_linux.conf build.conf
+fi
+
+sed -i -e "s:PREFIX:$PREFIX:g" build.conf
```

Comment:
I think inplace sed replacements are slightly different on Mac. Can you double check this please?

Reply:
It seems that "sed -i'' .." is the right way.

Reply:
There is however an error on Mac with above command, if I turn it back, at least the Mac build works fine.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=203061&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=2677

Reply:
@scopatz : The changed sed placement works today as well.